src/pages/partners/deriv-prime/why-deriv-prime/data.ts

```diff
@@ -0,0 +1,38 @@
+import { TString } from 'types/generics'
```

Comment:
Please move this file inside `features->pages->partners->deriv-prime->why-deriv-prime->data.ts`